scripts/get

```diff
@@ -48,25 +58,37 @@ if [ -n "$PKG_URL" -a -n "$PKG_SOURCE_NAME" ]; then
     sleep 1
   done
 
-  if ! [ -f $SOURCES/$1/$PKG_SOURCE_NAME -a "$(cat $STAMP 2>/dev/null)" == "$PKG_URL" ]; then
-    rm -f $SOURCES/$1/$PKG_SOURCE_NAME $STAMP
+  if ! _get_file_already_downloaded $1; then
```

Comment:
Missed that.
Wouldn't be code cleaner if **_get_file_already_downloaded** would be called again after flock loop and after this line content of if block.